### ISSUE TITLE
text blocks with escaped line terminators not handled correctly

### ISSUE THREAD

**Issue by errael** [eppleton/frgaal#8]
The jdk-18 is zulu, similar results with openjdk-17
Found this with frgaal 19.0.0-RC1, also fails with 18.0.0
Linix, all tests run in same terminal window.
All C.class files have major version: 62

Expect to see `'four five six '` on the same line. But with frgaal they are on separate lines and the escape char is still there.

```
$ cat C.java 
package play;

class C {

    public static void main(String[] args) {
        String s = """
            one
            two
            three
            four \
            five \
            six \
            """;
        System.out.println("'" + s + "'");
    }
}
```

```
$ /ref/openjdk/jdk-18/bin/javac C.java
$ mv C.class play
$ /ref/openjdk/jdk-18/bin/java -cp . play/C
'one
two
three
four five six '
```

```
$ type frgaal
frgaal is aliased to `java -jar /junk/Caches/maven_repository/org/frgaal/compiler/18.0.0/compiler-18.0.0.jar'
$ frgaal -source 18 -target 18 C.java
$ mv C.class play
$ /ref/openjdk/jdk-18/bin/java -cp . play/C
'one
two
three
four \
five \
six \
'
```

**Comment by jtulach** [comment]
Version `19.0.0` is out. Does it fix the problem?

**Comment by errael** [comment]
This report is about an issue with frgaal. A regular jdk works fine.

**Comment by errael** [comment]
It work, thanks. (I was a little confused because I didn't see the release on the github pages)

I was guessing this got missed because it's implemented as a hack in a front end parser.

**Comment by dukescript** [comment]
Fixed in version 19.0.0
